evaluate/task/task-transpile_test.go

```diff
@@ -546,3 +526,87 @@ func TestValidateTranspileRepository(t *testing.T) {
 		})
 	})
 }
+
+func TestTaskTranspileUnpackTranspilerPackage(t *testing.T) {
+	type testCase struct {
+		Name string
+
+		Before              func(packagePath string)
+		DestinationLanguage language.Language
+
+		RepositoryPath string
+		PackagePath    string
+
+		ExpectedOriginFilePathsWithLanguage map[string]language.Language
+		ExpectedStubFilePath                string
+	}
+
+	validate := func(t *testing.T, tc *testCase) {
+		t.Run(tc.Name, func(t *testing.T) {
+			logOutput, logger := log.Buffer()
+			defer func() {
+				if t.Failed() {
+					t.Logf("Logging output: %s", logOutput.String())
+				}
+			}()
+
+			temporaryDirectory := t.TempDir()
+
+			require.NoError(t, osutil.CopyTree(filepath.Join("..", "..", "testdata", tc.RepositoryPath), filepath.Join(temporaryDirectory, "testdata", tc.RepositoryPath)))
+
+			repository, cleanup, err := TemporaryRepository(logger, filepath.Join(temporaryDirectory, "testdata"), tc.RepositoryPath)
+			require.NoError(t, err)
+			defer cleanup()
+
+			if tc.Before != nil {
+				tc.Before(filepath.Join(repository.DataPath(), tc.PackagePath))
+			}
+
+			taskTranspile := TaskTranspile{}
+			ctx := evaltask.Context{
+				Language:   tc.DestinationLanguage,
+				Repository: repository,
+			}
+			actualOriginFilePathsWithLanguage, actualStubFilePath, actualErr := taskTranspile.unpackTranspilerPackage(ctx, logger, tc.PackagePath)
+			require.NoError(t, actualErr)
+
+			expectedOriginFilePathsWithLanguage := map[string]language.Language{}
+			for filePath, language := range tc.ExpectedOriginFilePathsWithLanguage {
+				expectedOriginFilePathsWithLanguage[filepath.Join(repository.DataPath(), filePath)] = language
```

Comment:
I guess in the implementation it does not matter cause the paths are absolute anyways. But here I noticed in the test cases below, that the `ExpectedOriginFilePath`s contain the package in the front, and the stub file path does not. Just a minor inconsistency but a bit confusing. If we change this to `filepath.Join(repository.DataPath(), tc.PackagePath, filePath)`, that should fix it.